app/app.go

```diff
@@ -176,8 +176,8 @@ func NewDesmosApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 		gov.StoreKey, upgrade.StoreKey, params.StoreKey, evidence.StoreKey,
 
 		// Custom modules
-		magpieTypes.StoreKey, postsTypes.StoreKey, profilesTypes.StoreKey, reportsTypes.StoreKey,
-		relationshipsTypes.StoreKey,
+		magpieTypes.StoreKey, postsTypes.StoreKey, relationshipsTypes.StoreKey,
+		profilesTypes.StoreKey, reportsTypes.StoreKey,
```

Comment:
There's no need here to change the order of store keys. They are actually initialized based on the order that the `AppModuleBasic` are placed in